go.mod

```diff
@@ -0,0 +1,16 @@
+module gopkg.in/AlecAivazis/survey.v1
```

Comment:
Do we still need the `v1` in our imports since this can be considered the first version of the package?

Reading [the page on the go repo](https://github.com/golang/go/wiki/Modules) makes it sound like you only have to have the version in the import path for major versions > 1

Reply:
I think it's required.  
Because the major version `v1` is contains in the import path: `import gopkg.in/AlecAivazis/survey.v1`.

It can use `module github.com/AlecAivazis/survey` without version, but user should use `replace` in go.mod
```
import gopkg.in/AlecAivazis/survey.v1 v1.6.4
replace gopkg.in/AlecAivazis/survey.v1 => github.com/AlecAivazis/survey v1.6.4
```

or direct import `github.com/AlecAivazis/survey` in gofile and go.mod
```
import github.com/AlecAivazis/survey v1.6.4
```

Reply:
Yea i guess i was suggesting that we remove it from the canonical import path. This wouldn't break people's current imports since gopkg.in would still resolve to the v1 releases

Reply:
I think there is an another problem:

Some golang files in survey project also use `import gopkg.in/AlecAivazis/survey.v1`, such as [core/renderer.go](https://github.com/AlecAivazis/survey/blob/master/core/renderer.go)

if we use `module gopkg.in/AlecAivazis/survey.v1`,these files will build failed.
In contrast, if we change these gofile's import path to `github.com/AlecAivazis/survey`, I guess the project used gopkg will get failure in run time

Reply:
After talking it over with @coryb i think the right move here is to go to `v2` with the `go.mod` file. This will keep everyone's imports working since gopkg.in will still resolve v1 to a version of the source with the `v1` references